Volume/Models/Publication.swift

```diff
@@ -8,35 +8,11 @@
 
 import Foundation
 
-// MARK: start of dummy data
-private let cremeDeCornell = Publication(
-    description: "Passionate food enthusiasts coming together to publish a diversity of recipes and stories.",
-    name: "Creme de Cornell",
-    image: "creme",
-    isFollowing: false,
-    recent: "Kale Chips"
-)
-
-private let cremeDeCornell2 = Publication(
-    description: "Passionate food enthusiasts",
-    name: "Creme",
-    image: "creme",
-    isFollowing: false,
-    recent: "Kale Chips"
-)
-
-let publicationsData = [
-    cremeDeCornell, cremeDeCornell2
-]
-// MARK: end of dummy data
-
-struct Publication: Hashable, Identifiable {
-    var id = UUID()
-    
+struct Publication: Codable, Hashable, Identifiable {
     let description: String
     let name: String
-    let image: String
+    let id: String
+    let imageUrl: URL?
     let isFollowing: Bool
```

Comment:
`ImageURL` or `ImageUrl`? I don't have a preference but you used `ImageURL` for articles